.ci/project-dependencies.yaml

```diff
@@ -0,0 +1,16 @@
+version: "2.1"
+dependencies:
+  - project: kiegroup/drools
+    mapping:
+      dependencies:
+        default:
+          - source: (\d*)\.(.*)\.(.*)
+            targetExpression: "process.env.GITHUB_BASE_REF.replace(/(\\d*)\\.(.*)\\.(.*)/g, (m, n1, n2, n3) => `${+n1-7}.${n2}.${n3}`)"
```

Comment:
Is the mapping correct?

Reply:
I was thinking something for drools like:

```
dependencies:
     default:
          - source: main
            target: main
```

and something for ansible-integration like:
```
dependencies:
     default:
          - source: main
            target: 8.35.x
```

Reply:
thanks @lampajr , let's wait for clarification from drools/ansible people @mariofusco 
ok?

Reply:
we agreed on having until versions and branches are clearer 
```
version: "2.1"
dependencies:
  - project: kiegroup/drools
    mapping:
      dependant:
        default:
          - source: 8.35.x
            target: 1.0.x

  - project: kiegroup/drools-ansible-rulebook-integration
    dependencies:
      - project: kiegroup/drools
    mapping:
      dependencies:
        default:
          - source: 1.0.x
            target: 8.35.x
```